messaging/methods/messages/getMessages.md

```diff
@@ -86,11 +83,8 @@ Link: <https://messaging.bandwidth.com/api/v2/users/{accountId}/messages?fromErr
 {
     "totalCount":1,
     "pageInfo": {
-        "before": 0,
-        "hasBefore": false,
-        "after": 100,
-        "hasAfter": false,
-        "limit": 100
+        "prevPage": "",
+        "nextPage": ""
```

Comment:
If no links are there then tags are not present (atleast this is how currently it is implemented). Something like this 
`
"pageInfo": {}
`

Reply:
Cool, updated. Also removed the links, didn't mean to have those there